jupyterhub_moss/templates/option_form.html

```diff
@@ -53,13 +53,24 @@
 
       <input
         type="radio"
-        id="mediumCore"
+        id="quarterCore"
         name="nprocs_simple"
         value="1"
       />
-      <label for="mediumCore">
+      <label for="quarterCore">
+        <p>&frac14&nbsp;node</p>
+        <p id="quarterCpufield" class="label-extra-info"></p>
+      </label>
+
+      <input
+        type="radio"
+        id="halfCore"
+        name="nprocs_simple"
+        value="1"
```

Comment:
Is `value` necessary here ? It will be overwritten by the JS part anyway, no ?

Reply:
I don't recall why all fields have `value="1"`, it's most probably not needed.

Reply:
I removed them.